### ISSUE TITLE
Add disk usage and free space in bytes alongside percentages

### ISSUE THREAD

**Issue by ond000** [domalab/ha-unraid#28]
**Is your feature request related to a problem? Please describe.**
Not applicable

**Describe the solution you'd like**
Currently we have only percentage usage of disks, array, temp, cache, but I would like to have it in Bytes too. And free space for disks would be nice too (percentage for free space can be counted by helper, but accurate Bytes of free space is harder).

**Use Case or Why Do You Want This?**
Describe a specific scenario where this feature would be helpful or explain why you think it's necessary.

**Priority of Request**
How important is this feature to you?  
- \[ ] High
- \[ ] Medium
- \[x] Low

**Additional context**

**Comment by domalab** [comment]
I don't think I'll implement byte-level disk information in the integration, and here's why:

While it's technically possible to add these sensors, this detailed disk information is best managed directly in Unraid's interface.

The current percentage-based sensors in Home Assistant provide a good overview for monitoring purposes, while keeping the integration focused and efficient.

If you have a specific use case in mind for needing byte-level information in Home Assistant that I haven't considered, I'd be interested in hearing more about it.

**Comment by ond000** [comment]
Yeah, you're right, there is no urgent need to have this detailed information HA, peoples can handle this in Unraid GUI.
